content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+---
+title: 'Get Started: Experiments'
+---
+
+# Get Started with Experiments
+
+<abbr>Experiments</abbr> proliferate quickly in ML projects where there are many
+parameters to tune or other permutations of the code. We can organize such
+projects and keep only what we ultimately need with `dvc experiments`. DVC can
+track experiments for you so there's no need to commit each one to Git. This way
+your repo doesn't become polluted with all of them. You can discard experiments
+once they're no longer needed.
+
+Previously, we learned how to tune [ML pipelines](/doc/start/data-pipelines) and
+[compare the changes](/doc/start/metrics-parameters-plots). In this section, we
+will explore the basic features of DVC experiment management with
+[`get-started-experiments`][gse] project.
+
+[gse]: https://github.com/iterative/get-started-experiments
+
+## ⚙️ Installing the Example Project
```

Comment:
(disclaimer: I haven't participated actively in the discussion so far, so sorry if I repeat certain concerns/arguments)

I think second main objective we should be building these documents with- focus on main things first. In this case it's `dvc exp run` - I should be able to see it in the fist screen with a table. Everything else should come as expandable and/or after sections.

My point here is that get-started are not regular tutorials, they should have a bit different structure. Reproducibility can go in the Katakoda, for example.

Reply:
There are various aspects of this question:

- Being fully reproducible is better than hiding some commands I think. If someone doesn't know about `dvc pull` (or `tar`, or `git clone`...) and we don't mention here, that may be a blocker for them. ("This thing doesn't work!!") A _veteran_ can simply skip these but a novice may drown in unknown waters. I prefer to err on the side of novices in this section of the docs. 

- I agree with you hiding these in `details` section, and other versions had these hidden. In the discussion, I thought there is _partial consensus_ to move the preparation to the main text by simplifying it. We can discuss this here I believe. @dberenbaum @casperdcl @jorgeorpinel

Reply:
Updated the span of `details` in 9c324a060665f01de3c4210d9ab86551ab4c8088